### ISSUE TITLE
Probably stoped working

### ISSUE THREAD

**Issue by KonradLinkowski** [ewfian/How-to-create-newline-in-Github-Bio#3]

**Comment by KonradLinkowski** [comment]
` ` works instead

**Comment by sananta** [comment]
I was able to do it @ewfian 's way. You have to make sure you are editing your bio after clicking the "Settings" tab only and then pressing "Update" after you make your changes (not from your main profile area): 
<img width="1295" alt="Screen Shot 2019-08-02 at 6 49 47 PM" src="https://user-images.githubusercontent.com/25359509/62405763-723e2100-b556-11e9-9ac8-47b9a0c08964.png">

**Comment by KonradLinkowski** [comment]
Ok, so it seems like using `&nbsp;` on profile page is impossible now.

**Comment by sananta** [comment]
Yeah, I tried, it's not possible

**Comment by haroldao** [comment]
so... how to do it now?

**Comment by wiktorkowalski** [comment]
I've tried it right now, it works. You're probably doing something wrong, but i can't help you because i don't even know how it works or ever how i did it, just try and try.

**Comment by haroldao** [comment]
Yeah, I saw your profile and it worked for you ... 😯... I'll try it again!

**Comment by 0xhalil** [comment]
I did it this way.

![How to create new line in GitHub](https://user-images.githubusercontent.com/71381757/93329368-c9b85280-f825-11ea-8030-50d77ff1f755.png)

**Comment by 411A** [comment]
❌ not working on the profile page
✔ but works from https://github.com/settings/profile
i used **`&emsp;`** to insert 4 spaces instead of 1 space (**`&nbsp;`**).

**Comment by lechnerio** [comment]
can confirm @KonradLinkowski -` `- still works.

**Comment by ghost** [comment]
> ` ` works instead

Yep

**Comment by porobertdev** [comment]
This also works, straight from the bio on the main page, with no DevTools: https://github.com/orgs/community/discussions/18184#discussioncomment-6508764

But to be honest, it seem to look weird when resizing the window.
